lib/Lolifiers/Blackout.php

```diff
@@ -0,0 +1,36 @@
+<?php
+
+namespace Monolol\Lolifiers;
+
+
+use Monolol\Lolifier;
+
+class Blackout implements Lolifier
+{
+    const MESSAGE = ' ... hum ... i forgot the rest of this sentence';
+
+    public function isHandling(array $record)
+    {
+        return true;
+    }
+
+    public function lolify(array $record)
+    {
+        $record['message'] = $this->forgotHalfSentence($record['message']);
+
+        return $record;
+    }
+
+    private function forgotHalfSentence($message)
+    {
+        $nbWords = str_word_count($message);
+
+        if($nbWords > 1)
+        {
+            $posHalfSentence = floor(strlen($message) / 2);
+            $message = substr($message, 0, strpos($message, ' ', $posHalfSentence)) . self::MESSAGE;
```

Comment:
@Taluu 
`$str = strrev(strpbrk(strrev($str), " \t\n\r\0\x0B"));` This remove only the last word of the sentence. This way the message might keep too much information ;)

I prefer @jvengeon solution to keep only the first half of the sentence.
What do you think @devwebpeanuts ?